src/SFA.DAS.EmployerIncentives.Web/Controllers/HomeController.cs

```diff
@@ -19,15 +19,15 @@ public class HomeController : Controller
     {
         private readonly ILegalEntitiesService _legalEntitiesService;
         private readonly ExternalLinksConfiguration _configuration;
-        private const int NewAgreementVersion = 5;
+        private const int NewAgreementVersion = 6;
```

Comment:
I know this was part of another PR (sorry!) but this can't really be here, it needs to come from the inner API so all values can be centralised.  Ideally the API should just return a bool indicating whether the agreement has been signed or not.

Reply:
makes sense

Reply:
should this value come from a JSON config? 😁